### ISSUE TITLE
FEAT: Add field sort and optimize default value when editing and updating

### ISSUE THREAD

**Issue by b0123456789** [beekeeper-studio/beekeeper-studio#1114]
Can't set a empty string for string and Can't sort fied in mysql.

**Comment by rathboma** [comment]
Do you mean when editing a table schema?

**Comment by b0123456789** [comment]
Yes, I already find the way to set a emty string for string. I close this.

**Comment by b0123456789** [comment]
![5](https://user-images.githubusercontent.com/21169660/164950889-84164f6c-583f-4df1-81c6-6be061368607.png)
![6](https://user-images.githubusercontent.com/21169660/164950891-76132369-d08d-45b1-9e09-7c743afd498f.png)

**Comment by b0123456789** [comment]
I use "```''```" to set  a empty string for code.
then reopen editor, i  found the  defaut value to be ```(EMPTY)```,
it look strange,
i think displaying value should be same with inputting value,
it confuss me at first time.
